workers/go/throughputstress/workflow.go

```diff
@@ -138,19 +139,29 @@ func ThroughputStressWorkflow(ctx workflow.Context, params *throughputstress.Wor
 				return workflow.ExecuteLocalActivity(localActCtx, activityStub.SelfSignal, ASignal).Get(ctx, nil)
 			},
 			func(ctx workflow.Context) error {
-				// This activity can simulate different traffic patterns as it will sleep for
-				// a configured duration. The duration is randomly selected from a distribution.
-				actInput := MakeSleepInput(params.SleepActivityPerPriority)
-				if actInput == nil {
+				// This activity simulates custom traffic patterns by generating activities that sleep
+				// for a specified duration, with optional priority and fairness keys.
+				if params.SleepActivities == nil {
 					return nil
 				}
-				opts := defaultActivityOpts(ctx)
-				opts.Priority = temporal.Priority{
-					PriorityKey: int(actInput.Priority),
+				sleepInputs := params.SleepActivities.Sample()
+
+				var sleepFuncs []func(workflow.Context) error
+				for _, actInput := range sleepInputs {
+					sleepFuncs = append(sleepFuncs, func(ctx workflow.Context) error {
+						opts := defaultActivityOpts(ctx)
+						opts.Priority.PriorityKey = int(actInput.PriorityKey)
+						if actInput.FairnessKey != "" {
+							// TODO(fairness): hack until there is a fairness key in the SDK
+							opts.ActivityID = fmt.Sprintf("x-temporal-internal-fairness-key[%s:%f]-%s",
```

Comment:
This part might be contentious. I don't think anyone outside of the Fairness crew would even consider using this; so I think it's fine/safe.

It would make the Fairness crew's life easier if we could get this PR into main; but if there are strong objections here, we can find another way to run off the branch.